Taarafo.Core.Tests.Unit/Services/Foundations/Profiles/ProfileServiceTests.Logic.RetrieveAll.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Taarafo.Core.Models.Profiles;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Profiles
+{
+    public partial class ProfileServiceTests
+    {
+        [Fact]
+        public void ShouldRetrievAllProfiles()
```

Comment:
If the scope is for Add then RetrieveAll should be removed.

Reply:
@ShriHumrudha is it critical to remove this? Coz the next commit will be the code rub of all comments. I think @rcruzfreelance thought we could do crud in the same branch. It turns out to be PRs should have single responsibility. 
We would branch out from master if this gets merged for remaining cases of retrieve all and retrieve by id. 

Question: Can we merge this PR with this tests case as well? Next time we will be more careful with that...

Reply:
@ElbekDeveloper  the RetrieveAll was removed seeing that the purpose of the branch name is work with Add. I apologize in part.

Reply:
nvm @hassanhabib was able to explain it to me